pkg/controller/volume/persistentvolume/metrics/metrics.go

```diff
@@ -231,6 +247,18 @@ func (collector *pvAndPVCCountCollector) pvcCollect(ch chan<- metrics.Metric) {
 	}
 }
 
+// RecordRetroactiveStorageClassMetric increments only retroactive_storageclass_total
+// metric or both retroactive_storageclass_total and retroactive_storageclass_errors_total
+// if success is false.
+func RecordRetroactiveStorageClassMetric(success bool) {
```

Comment:
I don't see this function called anywhere

Reply:
Resolved, silly git mistake. It's used in PV controller.